docs/user-guide/configuration/coverage.md

```diff
@@ -0,0 +1,47 @@
+---
+layout: main
+title: Coverage
+category: User Guide
+menu: menu
+toc:
+    - title: Coverage
+      url: "#coverage"
+---
+# Coverage
+
+Builds can upload coverage data after they are run.
+We currently support [SonarQube](https://github.com/screwdriver-cd/coverage-sonar) for coverage bookends.
+
+## SonarQube
+
+To use SonarQube, add a `sonar-project.properties` file in the root of your source code and add configurations there.
+
+Example `sonar-project.properties` file:
+```
+sonar.sources=lib
+sonar.javascript.lcov.reportPath=artifacts/coverage/lcov.info
+```
+
+Alternatively, you can add configurations to the environment variable `$SD_SONAR_OPTS`.
```

Comment:
We shall mention that if they have defined same property in both way, $SD_SONAR_OPTS will override the properties file